test_scripts/get_data_from_csv_datasource.sh

```diff
@@ -0,0 +1,4 @@
+source vars.txt
+echo  $SEARCHENGINE_IMAGE
+sudo docker run -v /data/searchengine/searchengine/:/etc/searchengine/  -v /data/searchengine/searchengine/logs/:/opt/app-root/src/logs/ --network searchengine-net $SEARCHENGINE_IMAGE get_index_data_from_csv_files -d test_csv  -f  /etc/searchengine/test_images.csv -r image -u False
```

Comment:
-d should ``NEW_DATA_SOURCE``
Also ``/etc/searchengine/test_images.csv`` will depend on the data source used